samples/data_versioning_mlflow_many_dependent_models/README.md

```diff
@@ -0,0 +1,59 @@
+For infrastructure setup, data preparation, please refer to the [DataOps Github Repository](https://github.com/Azure-Samples/modern-data-warehouse-dataops/e2e_samples), under directory `dataset_versioning`.
+
+# Overview
+
+The sample *Databricks* notebook demonstrates how to use MLFlow to track data versioning with DeltaLake and how to track many dependent models using MLFlow. It is developed based on the [Databricks sample notebook](https://docs.databricks.com/_static/notebooks/mlflow/mlflow-delta-training.html) using [Lending Club Loan Dataset hosted on Kaggle](https://www.kaggle.com/husainsb/lendingclub-issued-loans?select=lc_loan.csv). We provide an HTML version for a quick preview.
+
+__What does this sample demonstrate__:
+
+* Track ML experimentation and model performance using MLFlow on Databricks, and trace back to the training dataset version.
+* Access data using credential passthrough on Databricks (note, Databricks Premium is required).
+* Use delta format tabular data for training.
+* Manage dependent models using MLFlow, in this sample, a logistic regression and its dependent LIME model.
+
+__What doesn't this sample demonstrate__:
+
+* Comprehensive unit and integration tests - this sample doesn't have the level of unit tests or integration tests adequate for production deployment yet. 
+* Production scale inferencing - this sample stops at experimentation level.
+* End-to-end data and infrustructure pipelines. Please refer to `dataset_versioning` in [DataOps Github Repository](https://github.com/Azure-Samples/modern-data-warehouse-dataops/tree/master/e2e_samples) for the end-to-end DataOps process, including infrastructure setup, data pipeline setup, data copying and security setup etc.
+
+**Pipeline Structure**  
+This example assumes a scenario that experiments are conducted on Databricks using delta format data. It also assumes that more than one model is required in the experiments and the models are dependent. In this case, it shows how to train a logistic regression model and a LIME model to explain it.
```

Comment:
"scenario that experiments" => "scenario where experiments" or "scenario in which experiments"

Reply:
updated